Tools/autotest/quadplane.py

```diff
@@ -1568,6 +1568,7 @@ def ShipLanding(self):
             "Q_WP_SPEED": 700,
             "SIM_SHIP_DSIZE": 10,
             "FOLL_ENABLE": 1,
+            "FOLL_DIST_MAX": 1000,
```

Comment:
we should discuss if we should require this. The test passes without it, but with a lot of "No target location/velocity" messages. Previously the APIs that ship landing used didn't depend on FOLL_DIST_MAX being set

Reply:
maybe default FOLL_DIST_MAX to 0 on plane? and make 0 mean no limit?